quarkus-otel-db-perf/src/main/java/com/brunobat/rest/resource/LegumeApi.java

```diff
@@ -111,7 +111,7 @@ Response delete(
             @Parameter(name = "id",
                     description = "Id of the Legume to delete",
                     required = true,
-                    example = "81471222-5798-11e9-ae24-57fa13b361e1",
+                    example = "123",
                     schema = @Schema(description = "uuid", required = true))
```

Comment:
Nice catch - please teach me, did you notice this by chance or is there some tool that I could have run to tell me?